security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_migrate_config.php

```diff
@@ -58,84 +46,83 @@
 
 // Check the configuration version to see if XMLRPC Sync should
 // auto-disabled as part of the upgrade due to config format changes.
-if (empty($config['installedpackages']['snortglobal']['snort_config_ver']) && 
-    ($config['installedpackages']['snortsync']['config']['varsynconchanges'] == 'auto' ||
-     $config['installedpackages']['snortsync']['config']['varsynconchanges'] == 'manual')) {
-	$config['installedpackages']['snortsync']['config']['varsynconchanges']	= "disabled";
+if (empty(config_get_path('installedpackages/snortglobal/snort_config_ver')) && 
```

Comment:
Trailing whitespace here

Reply:
Didn't realize this was a thing, but removed it as suggested.

Reply:
Git usually complains (loudly, in some cases) about it. There may be more, sometimes it doesn't always print them all, after reporting a few it gives up. Some editors have a way to trim all trailing whitespace, too.